content/ja/docs/reference/access-authn-authz/rbac.md

```diff
@@ -0,0 +1,1109 @@
+---
+title: RBAC認可を使用する
+content_type: concept
+aliases: [/rbac/]
+weight: 70
+---
+
+<!-- overview -->
+Role Based Access Control(RBAC)は、組織内の個々のユーザーのRoleをベースに、コンピューターまたはネットワークリソースへのアクセスを制御する方法です。
+
+
+<!-- body -->
+RBAC認可は{{< glossary_tooltip text="API group" term_id="api-group" >}} `rbac.authorization.k8s.io`を使用して認可の決定を行い、Kubernetes APIを介して動的にポリシーを構成できるようにします。
```

Comment:
I suggest:
```suggestion
RBAC認可は{{< glossary_tooltip term_id="api-group" >}} `rbac.authorization.k8s.io`を使用して認可の決定を行い、Kubernetes APIを介して動的にポリシーを構成できるようにします。
```
so that if someone localizes the glossary term in the future, this page automatically updates.

Reply:
Thank you I'll take it.